test/cpp_extensions/cudnn_extension.cpp

```diff
@@ -0,0 +1,73 @@
+#include <torch/torch.h>
+#include <cuda.h>
+#include <torch/torch.h>
+#include <ATen/ATen.h>
+#include <ATen/TensorUtils.h>
+#include <ATen/cudnn/Descriptors.h>
+#include <ATen/cudnn/Exceptions.h>
+#include <ATen/cudnn/Handles.h>
+#include <torch/csrc/cuda/cuda_check.h>
+#include <torch/torch.h>
+#include <iostream>
+
+using namespace at;
```

Comment:
I would prefer for all these `using` directives to be removed. None of the other C++ extension tests have `using` directives (e.g. https://github.com/pytorch/pytorch/blob/master/test/cpp_extensions/extension.cpp). Since these are examples I point people at, I would prefer them to have the highest coding style possible, which these using directives go against

Reply:
Seems more verbose, but I'm fine with being explicit.